newIDE/app/public/index.html

```diff
@@ -61,5 +61,24 @@
         </p>
       </div>
     </noscript>
+    <script>
+      (function() {
+        // Apply background color for startup
+        var backgroundColors = {
+          Dark: '#252525',
+          Nord: '#2E3440',
+          'Solarized Dark': '#00212B'
+        };
+        var themeName = JSON.parse(localStorage.getItem("gd-preferences"))['themeName'];
+        if (!(themeName in backgroundColors)) {
+          console.log('Theme does not have a background color');
+          alert('Theme does not have a background color');
```

Comment:
Never use alert, just log something in the console but this is not worth bothering the user with.